xds/src/test/java/io/grpc/xds/CdsLoadBalancerTest.java

```diff
@@ -119,6 +121,34 @@ public LoadBalancer newLoadBalancer(Helper helper) {
     }
   };
 
+  private final LoadBalancerProvider fakeRoundRobinLbProvider = new LoadBalancerProvider() {
```

Comment:
Same for this one.

Reply:
This one is not using global registry. The test uses a new registry and registers a fake EDS lb provider, because the global registry registers the built-in EDS lb provider.

Reply:
Alright.